incubator/airflow/templates/configmaps.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "{{ template "airflow.fullname" . }}-env"
+data:
+  ## Force UTC timezone
+  TZ: Etc/UTC
+  ## Postgres DB configuration
+  POSTGRES_HOST: "{{ template "airflow.postgresql.fullname" . }}"
+  POSTGRES_DB: "{{ .Values.postgresql.postgresDatabase }}"
+  POSTGRES_USER: "{{ .Values.postgresql.postgresUser }}"
+  POSTGRES_PASSWORD: "{{ .Values.postgresql.postgresPassword }}"
```

Comment:
You should not put passwords in a ConfigMap. This should be a secret.

Reply:
todo after merge